### PR TITLE
simple navigation setup

### PR DESCRIPTION
**Pull request by andrewMacmurray**
+ set up the basic navigation structure, may need to add urlParser at some point but not sure yet
+ use elm live for live reload (during development)

### CODE REVIEW

**Comment by gabrielperales** [comment]
This looks good to me... why this is not merged yet? @andrewMacmurray @ivanmauricio

File: package.json [hunk 1]
```diff
@@ -17,8 +16,7 @@
     "url": "https://github.com/foundersandcoders/keep-the-lights-on/issues"
   },
   "homepage": "https://github.com/foundersandcoders/keep-the-lights-on#readme",
-  "devDependencies": {
-    "chokidar-cli": "^1.2.0",
-    "elm": "^0.17.1"
+  "dependencies": {
+    "elm": "^0.18.0"
```
Comment (gabrielperales) [inline package.json]: elm-live should be also added to dependencies, shouldn't it?

**Comment by gabrielperales** [comment]
I'm trying to run this branch, but it is raising an error

captura de pantalla 2017-02-05 a las 14 20 23

**Comment by gabrielperales** [comment]
Ok, I solved the problem. I had to reinstall all the elm packages with `elm-package install` to update `elm-lang/html`.

**Comment by andrewMacmurray** [comment]
hey @gabrielperales ! I think we did this a while ago and then forgot about it 😆 thanks for taking a look, will add elm live to the devDependencies

**Pull request merged**